[2025-04-21T23:37:17.996+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-21T23:37:18.054+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews_pipeline.normalize_and_clean_data manual__2025-04-21T23:36:55.641358+00:00 [queued]>
[2025-04-21T23:37:18.069+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews_pipeline.normalize_and_clean_data manual__2025-04-21T23:36:55.641358+00:00 [queued]>
[2025-04-21T23:37:18.079+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-21T23:37:18.091+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): normalize_and_clean_data> on 2025-04-21 23:36:55.641358+00:00
[2025-04-21T23:37:18.104+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=417) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-21T23:37:18.108+0000] {standard_task_runner.py:72} INFO - Started process 432 to run task
[2025-04-21T23:37:18.108+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews_pipeline', 'normalize_and_clean_data', 'manual__2025-04-21T23:36:55.641358+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpg8bjhp86']
[2025-04-21T23:37:18.109+0000] {standard_task_runner.py:105} INFO - Job 59: Subtask normalize_and_clean_data
[2025-04-21T23:37:18.158+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews_pipeline.normalize_and_clean_data manual__2025-04-21T23:36:55.641358+00:00 [running]> on host 95fe2406fe61
[2025-04-21T23:37:18.254+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews_pipeline' AIRFLOW_CTX_TASK_ID='normalize_and_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-21T23:36:55.641358+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-21T23:36:55.641358+00:00'
[2025-04-21T23:37:18.259+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-21T23:37:18.265+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-21T23:37:18.266+0000] {logging_mixin.py:190} INFO - Current task name:normalize_and_clean_data state:running start_date:2025-04-21 23:37:18.057349+00:00
[2025-04-21T23:37:18.273+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews_pipeline and current dag run status:running
[2025-04-21T23:37:18.278+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-21T23:37:18.300+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-04-21T23:37:18.435+0000] {warnings.py:112} WARNING - /opt/***/dags/storeData.py:197: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql("SELECT * FROM review.raw_reviews", conn)

[2025-04-21T23:37:18.654+0000] {logging_mixin.py:190} INFO - Data normalized and cleaned successfully.
[2025-04-21T23:37:18.655+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-21T23:37:18.663+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-21T23:37:18.663+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews_pipeline, task_id=normalize_and_clean_data, run_id=manual__2025-04-21T23:36:55.641358+00:00, execution_date=20250421T233655, start_date=20250421T233718, end_date=20250421T233718
[2025-04-21T23:37:18.685+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-21T23:37:18.688+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-21T23:37:18.688+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews_pipeline queued_at:2025-04-21 23:36:55.658570+00:00
[2025-04-21T23:37:18.690+0000] {logging_mixin.py:190} INFO - Task hostname:95fe2406fe61 operator:PythonOperator
[2025-04-21T23:37:18.727+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-21T23:37:18.773+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-21T23:37:18.796+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
